crates/events/src/lib.rs

```diff
@@ -0,0 +1,192 @@
+/*!
+ */
+#![deny(missing_docs, missing_debug_implementations)]
+
+use std::borrow::Cow;
+use wasm_bindgen::closure::Closure;
+use wasm_bindgen::{JsCast, UnwrapThrowExt};
+use web_sys::{AddEventListenerOptions, Event, EventTarget};
+
+///
+#[derive(Debug, Clone, Copy)]
+pub struct EventListenerOptions {
+    ///
+    pub capture: bool,
+
+    ///
+    pub once: bool,
+
+    ///
+    pub passive: bool,
+}
+
+impl EventListenerOptions {
+    ///
+    #[inline]
+    pub fn to_js(&self) -> AddEventListenerOptions {
```

Comment:
`to_raw`?

Reply:
I'm not sure that name makes sense: it isn't wrapping an existing type (which is what `as_raw` or `to_raw` would work great for), it's actually converting into a different type entirely.

Maybe it just shouldn't expose it at all, and we can defer the name decision to later (since `to_js` is mostly an implementation detail anyways).

Reply:
> Maybe it just shouldn't expose it at all, and we can defer the name decision to later (since `to_js` is mostly an implementation detail anyways).

:+1: